Sources/PackageDescription/PackageDescription.docc/PackageDescription.md

```diff
@@ -1,46 +1,50 @@
-    // swift-tools-version:5.0
-    // swift-tools-version:5.1
-    // ...
-    // swift-tools-version:5.6
-
-The Swift tools version declares the version of the `PackageDescription` library, the minimum version of the Swift tools and Swift language compatibility version to process the manifest, and the minimum version of the Swift tools that are needed to use the Swift package. Each version of Swift can introduce updates to the `PackageDescription` framework, but the previous API version will continue to be available to packages which declare a prior tools version. This behavior lets you take advantage of new releases of Swift, the Swift tools, and the `PackageDescription` library, without having to update your package's manifest or losing access to existing packages.
+Create reusable code, organize it in a lightweight way, and share it across your projects and with other developers.
+
+Swift packages are reusable components of Swift, Objective-C, Objective-C++, C, or C++ code that developers can use in their projects. They bundle source files, binaries, and resources in a way that’s easy to use in your app’s project. 
+
+Each Swift package requires a `Package.swift` file in the main directory of the package — referred to as the package manifest. When you create a Swift package, you use the PackageDescription library in the package manifest to list dependencies, configure localized resources, and set other configuration options.
+
+For example, the package manifest from the [SlothCreator: Building DocC Documentation in Xcode](https://developer.apple.com/documentation/xcode/slothcreator_building_docc_documentation_in_xcode) sample project below defines the SlothCreator package, with the SlothCreator library in it. It specifies the deployment targets, and that its resources are in the `Resources` folder.
+
+```swift
+import PackageDescription
+
+let package = Package(
+    name: "SlothCreator",
+    platforms: [
+        .macOS(.v11),
+        .iOS(.v14),
+        .watchOS(.v7),
+        .tvOS(.v13)
+    ],
+    products: [
+        .library(
+            name: "SlothCreator",
+            targets: ["SlothCreator"]
+        )
+    ],
+    targets: [
+        .target(
+            name: "SlothCreator",
+            resources: [
+                .process("Resources/")
+            ]
+        )
+    ]
+)
+```
+
+The package manifest also allows you to define executable products, as well as plugins that Swift Package Manager can use to build other products in the manifest.
+
+For more information about adding a package dependency to your app project and creating Swift packages with Xcode, see [Adding Package Dependencies to Your App](https://developer.apple.com/documentation/xcode/adding_package_dependencies_to_your_app), [Creating a Standalone Swift Package with Xcode](https://developer.apple.com/documentation/xcode/creating_a_standalone_swift_package_with_xcode/), and [Swift Packages](https://developer.apple.com/documentation/xcode/swift_packages).
```

Comment:
I think it's best not to rely on any website or example that's closed-source or not hosted by swift.org, unless there is no other choice.

Reply:
This paragraph is about working with Xcode, which I only mention to direct the reader to the correct place where they can read up on using Swift packages with Xcode, which is the Xcode documentation. This documentation is maintained by Apple, and bug reports are appreciated if they do become outdated. Since it's not desired to teach the use of Xcode in the open source documentation, linking to those documents is unavoidable.